### PR TITLE
[follows from #104] Wrap-up --local flow

### LINKED ISSUE #67: server --local should run in Flask development mode
The environment variable: `export FLASK_ENV=development` should be set somehow when running `server --local`. This allows the debugger and other niceties to be there for developers.

### LINKED ISSUE #72: assets command: split npm and assets building
Currently, it is using `webpack buildall` from `flask-webpackext`. It would be nice to split the npm installation from the assets building (e.g. add npm flag?).

### LINKED ISSUE #73: update command: should update all statics, not hardcode the `logo`
Currently the `update` command copies to the venv or the container volume the logo file, with a hardcoded name to a hardcoded path. In order to allow further customization it should read the contents (e.g. list) and update all accordingly.

### PR DESCRIPTION
**Pull request by fenekku**
Tomorrow I should be able to do most of the container work and removal of old commands / dependencies. Might also rename some stuff then.

- Closes #67 

The following were closed by prior PRs:
- Closes #72 
- Closes #73

### CODE REVIEW

**Comment by fenekku** [comment]
Sent a PR to flask-babelex https://github.com/mrjoes/flask-babelex/pull/19 to fix issue we are seeing on Travis (werkzeug is breaking imports for everyone)

File: invenio_cli/helpers/commands.py [hunk 1]
```diff
@@ -141,6 +144,108 @@ def build(self, pre, lock):
...
+        # TODO: Find faster way to procede when containers are ready
+        time.sleep(30)  # Give time to the containers to start properly
+
+    def services(self, force):
+        """Local start of containers (services).
+
+        NOTE: We use check=True to mimic set -e from original setup script
+              i.e. if a command fails, an exception is raised
+        """
+        self._containers_are_running()
```
Comment (zzacharo) [inline invenio_cli/helpers/commands.py]: Minor: Probably a bit more descriptive name would be `ensure_containers_running` or `start_and_wait_containers`?

File: invenio_cli/helpers/commands.py [hunk 2]
```diff
@@ -141,6 +144,108 @@ def build(self, pre, lock):
...
+        signal.signal(signal.SIGINT, signal_handler)
+
+        click.secho("Starting celery worker...", fg="green")
+        worker = subprocess.Popen([
+            'pipenv', 'run', 'celery', 'worker', '--app', 'invenio_app.celery'
+        ])
+
+        click.secho("Starting up local (development) server...", fg='green')
+        run_env = os.environ.copy()
+        run_env['FLASK_ENV'] = 'development'
```
Comment (zzacharo) [inline invenio_cli/helpers/commands.py]: Nice!

File: invenio_cli/helpers/commands.py (hunk 1 above)
Comment (fenekku) [reply invenio_cli/helpers/commands.py]: `ensure_containers_running` :+1:

**Pull request merged**